hw/ip/alert_handler/data/alert_handler.hjson

```diff
@@ -1488,6 +1491,7 @@
           enum: [
                   { value: "0b000", name: "Idle",     desc: "No timeout or escalation triggered." },
                   { value: "0b001", name: "Timeout",  desc: "IRQ timeout counter is active." },
+                  { value: "0b010", name: "FsmError", desc: "Terminal error state if FSM has been glitched." },
```

Comment:
@cindychip we may need to extend the alert handler tests to poke this FSM into an invalid state to test this.

Reply:
Thanks for tagging me, yup should be added to the testplan.